test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -443,10 +442,12 @@ UCS_TEST_P(test_ucp_tag_match, rndv_req_exp, "RNDV_THRESH=1048576") {
     EXPECT_TRUE(my_recv_req->completed);
     EXPECT_EQ(sendbuf, recvbuf);
 
-    EXPECT_TRUE(my_send_req->completed);
-    EXPECT_EQ(UCS_OK, my_send_req->status);
+    if (my_send_req != NULL) {
+        EXPECT_TRUE(my_send_req->completed);
```

Comment:
should not we wait for my_send_req completion here as well?
also maybe create a function-member for checking my_send_req as it is done in several places.